### PR TITLE
added s in font drpepper

### PR DESCRIPTION
**Pull request by bhaskar962**
added s in font drpepper

### Referenced Issues

closes #987

### CODE REVIEW

File: Fonts/drpepper/drpepper.h [hunk 1]
```diff
@@ -503,23 +506,26 @@ class Drpepper : public Fonts
...
+        char **character = getCharGrid(5,5);
+        character[0][0] = character[0][1] = character[0][2] = character[0][3] = character[1][0] = ' ';
+character[1][1] = character[1][2] = character[1][3] = character[2][1] = character[3][1] = character[3][2] = '_';
+character[2][0] = character[2][3] = '<';
+character[2][2] = '-';
+character[3][0] = character[3][3] = '/';
+        return character;
+    }
 
     */
```
Comment (AnshumanMahato) [inline Fonts/drpepper/drpepper.h:520]: remove this line... this is causing error

File: Fonts/drpepper/drpepper.h [hunk 2]
```diff
@@ -273,14 +275,15 @@ class Drpepper : public Fonts
...
-    {
-        char **character = getCharGrid(4, 5);
-        // Enter the character grid in ROWS X COLS
-        character[0][1] = character[0][3] = character[3][2] = '_';
-        character[1][0] = character[1][2] = character[1][4] = character[3][1] = character[3][3] = '|';
-        character[2][4] = '/';
-        character[2][0] = '\\';
+     char **Y()
+        {
+            char **character = getCharGrid(rows,cols);
```
Comment (AnshumanMahato) [inline Fonts/drpepper/drpepper.h:280]: remove this line... this is causing error

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]

**Comment by bhaskar962** [comment]
Now it's not showing any error

**Comment by AnshumanMahato** [comment]
did you push the changes? No new commits are shown here.

**Review by AnshumanMahato — APPROVED** [review]

**Pull request merged**